src/main/utils/_initialSettings.js

```diff
@@ -3,4 +3,5 @@ export default {
   enableJSONApi: true,
   nativeFrame: (process.platform !== 'win32'),
   enableWindowTitleInfo: false,
+  auto-launch: false,
```

Comment:
Ok, going to do this locally and test it again. didnt used 'npm run lint' before.